<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeTerminalEditor</class>
 <widget class="QWidget" name="FreeTerminalEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>279</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="m_move_label">
     <property name="text">
      <string>Déplacer dans :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="m_move_in_cb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="QTableView" name="m_table_view">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Fonction :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>LED :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="m_move_pb">
     <property name="toolTip">
      <string>Appliquer le déplacement</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../qelectrotech.qrc">
       <normaloff>:/ico/22x22/dialog-ok.png</normaloff>:/ico/22x22/dialog-ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QComboBox" name="m_type_cb">
     <item>
      <property name="text">
       <string>Générique</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fusible</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sectionnable</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Diode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Terre</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QComboBox" name="m_function_cb">
     <item>
      <property name="text">
       <string>Générique</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neutre</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QComboBox" name="m_led_cb">
     <item>
      <property name="text">
       <string>Sans</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Avec</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../qelectrotech.qrc"/>
 </resources>
 <connections/>
</ui>
